src/middlewares/bearer.js

```diff
@@ -1,6 +1,7 @@
 module.exports = (req, res ,next) => {
-	if (!req.headers.auth) {
-		req.authenticated = false;
+  console.log(req.headers);
```

Comment:
This `console.log` should be dropped if you want to push it live. It will generate unneeded logging on the Heroku console, in addition to just not looking pretty.